src/components/picker/nimble-picker.js

```diff
@@ -439,8 +596,8 @@ export default class NimblePicker extends React.PureComponent {
   handleKeyDown(e) {
     let handled = false
 
-    switch (e.keyCode) {
-      case 13:
+    switch (e.key) {
+      case 'Enter':
```

Comment:
Just because I'm not sure: Currently (with <kbd>tab</kbd> focus switches) also <kbd>space</kbd> works for selecting emojis. This here only refers to <kbd>Enter</kbd>.
IMHO <kbd>space</kbd> should also continue to work… :thinking: